autogen/agentchat/conversable_agent.py

```diff
@@ -78,6 +78,7 @@ def __init__(
         default_auto_reply: Union[str, Dict] = "",
         description: Optional[str] = None,
         chat_messages: Optional[Dict[Agent, List[Dict]]] = None,
+        is_silent: Optional[bool] = None,
```

Comment:
should this be called `silent` for consistency with `initiate_chat`?

Reply:
@umermansoor should be OK now?